src/test/java/com/wikia/webdriver/elements/mercury/pages/SearchResultsPage.java

```diff
@@ -0,0 +1,74 @@
+package com.wikia.webdriver.elements.mercury.pages;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.mercury.components.Loading;
+import com.wikia.webdriver.elements.mercury.components.Search;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import lombok.Getter;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class SearchResultsPage extends WikiBasePageObject {
+  @FindBy(css = ".search-results")
+  private WebElement searchResultsContainer;
+
+  @FindBy(css = ".search--no-results")
+  private WebElement noResultsContainer;
+
+  @Getter
+  private final Search search = new Search();
+
+  private String searchResultClass = ".search-results__list .search-results__item";
+
+  public SearchResultsPage openForQuery(String query) {
+    getUrl(String.format("%s%s", urlBuilder.getUrlForWiki(),
+                         URLsContent.MOBILE_SEARCH_RESULTS_PAGE.replace("%query%", query)));
+
+    return this;
+  }
+
+  public Search typeInSearch(String text) {
+    return this.getSearch().typeInSearch(text);
+  }
+
+  public SearchResultsPage selectSearchResult(int index) {
+    Loading loading = new Loading(driver);
+    String oldUrl = driver.getCurrentUrl();
+
+    PageObjectLogging.logInfo("Select search result no.: " + index);
+    WebElement searchResult = driver.findElements(By.cssSelector(searchResultClass)).get(index);
+    wait.forElementClickable(searchResult);
+    searchResult.click();
+    loading.handleAsyncPageReload();
+
+    Assertion.assertFalse(oldUrl.equalsIgnoreCase(driver.getCurrentUrl()),
+                          "Navigation to selected search suggestion failed");
+    PageObjectLogging.logInfo("Successfully navigated to selected search result");
+
+    return this;
+  }
+
+  public boolean isSearchResultsPageOpen() {
+    try {
+      wait.forElementVisible(searchResultsContainer);
+      return true;
+    } catch (NoSuchElementException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)